packages/firestore/test/unit/local/web_storage_shared_client_state.test.ts

```diff
@@ -206,22 +275,23 @@ describe('WebStorageSharedClientState', () => {
         persistenceHelpers.TEST_PERSISTENCE_PREFIX
       }_${AutoId.newId()}`;
 
-      const oldState = new LocalClientState();
-      oldState.addQueryTarget(5);
-
-      const updatedState = new LocalClientState();
-      updatedState.addQueryTarget(5);
-      updatedState.addQueryTarget(6);
-
       // The prior client has one pending mutation and two active query targets
       verifyState(1, [3, 4]);
 
+      const oldState = new LocalClientState();
+      oldState.addQueryTarget(5);
+
       storageCallback({
         key: secondaryClientKey,
         storageArea: window.localStorage,
         newValue: oldState.toLocalStorageJSON()
       });
-      verifyState(0, [3, 4, 5]);
+      verifyState(1, [3, 4, 5]);
+
+      const updatedState = new LocalClientState();
+      updatedState.addQueryTarget(5);
+      updatedState.addQueryTarget(6);
+      updatedState.addPendingMutation(0);
```

Comment:
I don't fully understand why you added this `addPendingMutation(0)` call to this test...  It seems like the old code was broken (it was returning 0 for getMinimumGlobalPendingMutation(), but it probably should have been returning -1 (BATCHID_UNKNOWN)... or maybe null).